Firestore/Protos/CMakeLists.txt

```diff
@@ -77,16 +77,6 @@ cc_library(
     cpp/google/firestore/v1beta1/query.pb.h
     cpp/google/firestore/v1beta1/write.pb.cc
     cpp/google/firestore/v1beta1/write.pb.h
-    cpp/google/protobuf/any.pb.cc
```

Comment:
This change also includes the removal of the libprotobuf-generated well-known types. Is that intended?

Reply:
Yes. Note the `rm -rf cpp/google/protobuf` line in build-protos.sh:33.

This patch makes these unnecessary. (I don't think the .cc files were ever necessary since the tests were being linked with libprotobuf, which includes them. But the .h files were being used due to -I order.)

Arguably, this should be a separate commit. I'll split 'em if you want. Otherwise, I'll update the commit message accordingly.

This also means that #1365 becomes obsolete.

Reply:
Happy to have these together--just wanted to make sure it's intentional.

Please exempt these from style checks (by adding `\%\.nanopb\.% d` to `scripts/style.sh`) and go for it.